generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
I have another suggestion I'd like to propose here. Again, the reason I haven't done it right away is because I wanted to change as few things as possible. However, if I were to allow myself to make more changes, I would move all assets that are meant to be embedded inside a single subdirectory, for example named `assets`.

Then **all** of this code:

```
ignore := func(fi os.FileInfo, path string) bool {
	switch {
	case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
		return true
	case pathpkg.Ext(fi.Name()) == ".go":
		return true
	case path == "/README.md":
		return true
	default:
		return false
	}
}
var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
```

Could be replaced with just:

```Go
var data http.FileSystem = http.Dir("assets")
```

Would you like this to be done?